### ISSUE TITLE
Academic Advising content accessibility audit

### ISSUE THREAD

**Issue by Computerrat** [phsc-edu/2018#334]
### Please review the following links for accessibility problems.

https://advising.phsc.edu/help/faq/advising-and-registration/can-i-obtain-my-degree-online
![can i obtain my degree online_ _ advising - google chrome 2018-08-06 12 26 49](https://user-images.githubusercontent.com/31025634/43730261-ceef0db0-9978-11e8-9ef6-e1acc65b6a2f.png)

--------------

https://advising.phsc.edu/help/faq/wise/i-have-forgotten-my-pin-what-should-i-do
![i have forgotten my pin what should i do_ _ advising - google chrome 2018-08-06 12 33 36](https://user-images.githubusercontent.com/31025634/43730307-ee93fe78-9978-11e8-8d97-81fad4179f3c.png)

**Comment by Computerrat** [comment]
As a side note, I found these two FAQs that did not have a link to WISE. As other FAQs did allow you to link to WISE, as a student, I found that a bit confusing.

https://advising.phsc.edu/help/faq/registration/how-do-i-change-my-schedule
![how do i change my schedule_ _ advising - google chrome 2018-08-06 12 41 13](https://user-images.githubusercontent.com/31025634/43730354-11dcdc10-9979-11e8-8f58-f579920ba483.png)

----------------

https://advising.phsc.edu/help/faq/registration/i-am-new-student-phsc-and-have-not-yet-registered-classes-can-i-register
![i am a new student at phsc and have not yet registered for classes can i register online_ _ advising - google chrome 2018-08-06 12 42 24](https://user-images.githubusercontent.com/31025634/43730397-3521d734-9979-11e8-8210-249a66a871ce.png)

**Comment by C13L0** [comment]
@curtiskphsc Please correct the two accessibility issues as documented above.

**Comment by curtiskphsc** [comment]
@C13L0 @Computerrat As I will need to communicate to our stakeholders the reason behind the request to have them modify their content, we need to have some clearly defined reasons, and we should be speaking the exact same language so as not to confuse anyone. Also, we should be very specific as to _when_, and _in what circumstances_, capitalization _is_ allowed. I am not an ADA compliance expert. "WISE" is acceptable?, "PHSC" is acceptable?, "STEM" is acceptable? We need to be _very_ clear on this. For example, on this page I believe we made Rick Casey change ACCUPLACER to sentence case Accuplacer https://testing-services.phsc.edu/tests/accuplacer when it is branded and widely known in higher ed as ACCUPLACER as demonstrated here https://accuplacer.collegeboard.org/?navId=gf-accu&navId=gf-accu there isn't one instance of the test name on this page where it is referenced in sentence case.
So, we need a formal guideline to reference when stating "This must be in sentence case and not in all caps" and we need to define the use of all caps so we can clearly state "You can _only_ use all caps in these circumstances...."
Some guidance on this language would be appreciated.

**Comment by C13L0** [comment]
@curtiskphsc @boettcc 

It is cut and dry. An acronym can be in all caps. If it is not an acronym, it cannot be all caps.

**Comment by curtiskphsc** [comment]
@C13L0 We still need a point of reference. A URL to direct them to or something.

**Comment by C13L0** [comment]
@curtiskphsc @boettcc The accessibility checker is used to check the reading order. It cannot decipher between a word in all caps or an acronym in all caps. I will be happy to remove the accessibility checker from all websites if you feel that it's confusing.

**Comment by C13L0** [comment]
@curtiskphsc @boettcc Please correct the two accessibility issues as documented above.

**Comment by curtiskphsc** [comment]
@C13L0 @boettcc I feel it is confusing, but if it is a useful tool we should keep it with an explanation. Everyday users may assume that using this checking tool makes their content compliant and will likely state as much. Corrected issues on pages as documented.

**Comment by C13L0** [comment]
@Computerrat Please review changes for accessibility and close ticket once verified.

**Comment by Computerrat** [comment]
Verified changes
